SpatialGDK/Source/SpatialGDK/Private/SpatialView/SubView.cpp

```diff
@@ -27,31 +27,6 @@ FSubView::FSubView(const Worker_ComponentId InTagComponentId, FFilterPredicate I
 	RegisterRefreshCallbacks(DispatcherRefreshCallbacks);
 }
 
-void FSubView::TagQuery(Query& QueryToTag) const
```

Comment:
Seems unrelated. I guess this is just some clean up that slides into this PR?

Reply:
yeah. Turned out this logic was a little overkill when actually trying to use subviews as part of this PR.